config/config.exs

```diff
@@ -32,6 +32,8 @@ config :membrane_rtc_engine_timescaledb, repo: VideoRoom.Repo
 
 config :membrane_videoroom_demo, ecto_repos: [VideoRoom.Repo]
 
-config :logger, :console, metadata: [:room, :peer]
+config :logger, :console,
+  format: "\n$time $metadata[$level] $message",
+  metadata: [:file, :line, :room_id, :rtc_engine, :webrtc_endpoint]
```

Comment:
I noticed that e.g. borwsers log also file and line which is very useful for debugging. I also propose removing new line at the end of each log so they are more concise. Example output will look like this

```
22:00:09.880 file=lib/videoroom/room.ex line=219 room_id=test [info] Engine terminated.
22:00:09.880 file=lib/videoroom/room/monitor.ex line=13 [info] Room: test terminated with reason: :normal
22:00:09.880 file=lib/videoroom_web/peer_channel.ex line=47 [info] Room died when trying to join. Creating a new one.
22:00:09.880 file=lib/videoroom/room.ex line=42 room_id=test [info] Spawning room process: #PID<0.1004.0>
22:00:09.880 file=lib/membrane_rtc_engine/engine.ex line=273 room_id=test [info] Starting a new RTC Engine instance with id: test
22:00:09.880 file=lib/videoroom/room.ex line=120 room_id=test [info] New peer: "daa7763f-edac-4bc8-8d19-320ca865b8b0". Accepting.
22:00:09.880 file=lib/phoenix/logger.ex line=374 [info] JOINED room:test in 21ms
  Parameters: %{"isSimulcastOn" => true}
22:00:10.392 file=lib/membrane_rtc_engine/engine.ex line=662 rtc_engine=test [info] [pipeline@<0.1006.0>] New incoming H264 track daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded (variant: low) from endpoint "daa7763f-edac-4bc8-8d19-320ca865b8b0"
22:00:10.403 file=lib/membrane_rtc_engine/tees/tee.ex line=165 rtc_engine=test [info] [{:tee, "daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded"}] Track variant {"daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded", :low} resumed.
22:00:10.698 file=lib/membrane_rtc_engine/engine.ex line=662 rtc_engine=test [info] [pipeline@<0.1006.0>] New incoming H264 track daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded (variant: high) from endpoint "daa7763f-edac-4bc8-8d19-320ca865b8b0"
22:00:10.700 file=lib/membrane_rtc_engine/tees/tee.ex line=165 rtc_engine=test [info] [{:tee, "daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded"}] Track variant {"daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded", :high} resumed.
22:00:10.703 file=lib/membrane_rtc_engine/engine.ex line=662 rtc_engine=test [info] [pipeline@<0.1006.0>] New incoming H264 track daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded (variant: medium) from endpoint "daa7763f-edac-4bc8-8d19-320ca865b8b0"
22:00:10.704 file=lib/membrane_rtc_engine/tees/tee.ex line=165 rtc_engine=test [info] [{:tee, "daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded"}] Track variant {"daa7763f-edac-4bc8-8d19-320ca865b8b0:046d4f3f-4836-404a-b034-ed8cd4267ded", :medium} resumed.
22:00:11.922 file=/home/michal/Repos/membrane_videoroom/deps/opentelemetry/src/otel_exporter.erl line=45 [info] OTLP exporter successfully initialized
```

Reply:
Wouldn't it be better to remove the new line from the line beginning? In the case of standard, buffered output lack of `\n` will postpone the log printing.

The idea with the file is great, I like it :+1: